tests/MongoDB.Driver.Tests/MongoClientSettingsTests.cs

```diff
@@ -439,8 +453,8 @@ public void TestFromUrl()
             Assert.True(url.Servers.SequenceEqual(settings.Servers));
             Assert.Equal(url.ServerSelectionTimeout, settings.ServerSelectionTimeout);
             Assert.Equal(url.SocketTimeout, settings.SocketTimeout);
-            Assert.Equal(null, settings.SslSettings);
 #pragma warning disable 618
+            settings.SslSettings.Should().Be(new SslSettings {CheckCertificateRevocation=url.VerifySslCertificate});
```

Comment:
space after `{` and before `}`. Use Ctrl + K + D to make this formatting automatically (at least in vs) :)

Reply:
Fixed! :)